cmd/root.go

```diff
@@ -30,12 +30,13 @@ import (
 
 var (
 	rootCmd = &cobra.Command{
-		Use:   dist.AnalyzeExec,
-		Short: fmt.Sprintf("A CLI to analyze artifacts from a %s", dist.ServerProduct),
-		Long: fmt.Sprintf(`Analyze your %s artifacts to understand the effort to upgrade
-your infrastructure by generating reports, automatically fixing violations
-and/or deprecations, and generating Effortless packages.
-`, dist.ServerProduct),
+		// This is intended to be used as part of Workstation, via the `chef` command
+		// but will also support other wrappers via dist.CLIWrapperExec.
+		Use:   fmt.Sprintf("%s report", dist.CLIWrapperExec),
```

Comment:
It feels weird that running `chef-analyze` outputs
```
Reports to assist you in understanding the effort to
upgrade your infrastructure to latest patterns and practices.
Can aggregate information by nodes or cookbooks.

Usage:
  chef report [flags]
  chef [command]

Available Commands:
  capture     Capture a node's state into a local chef-repo
  help        Help about any command
  report      Generate reports from a Chef Infra Server

Flags:
  -h, --help   help for chef

Use "chef [command] --help" for more information about a command.
```

I would have prefered a no-arg-no-flag `chef-analyze` execution to output something like `"Upgrade Lab is used via the 'chef' command"`. I tried `rootCmd.SetHelpTemplate` but that sets the help template for all the children too, which screws up `chef report`.

Reply:
Actually I think if you change line 49 from `rootCmd.Help()` to `fmt.Println("Upgrade Lab is meant to be used from the 'chef' executable")` it will achieve what I was thinking.

Since `chef-analyze` is not meant to be used by end-users I don't see the benefit of adding `Use`, `Short`, `Long`, etc. to the root command. Thoughts?

Reply:
Yep, that sounds good - the other changes here also make it less-friendly to be used standalone (eg usage text that now always shows `chef report` for the report commands).

Reply:
Found a way to make it mostly never show the command usage. It will show the wrong help if you specify `chef-analyze help bad-command-name` but other help cases (chef-analyze, chef-analyze help, chef-analyze -h|--help)  will show the 'use chef executable` message.